docs/controllers/leaderelection.md

```diff
@@ -0,0 +1,55 @@
+# Leader Election
+
+This chapter is a stub about Leases, Leader Election and how to achieve distributed locks around Kubernetes resources.
+
+!!! note "Terminology"
+
+    See [Kubernetes//Leases](https://kubernetes.io/docs/concepts/architecture/leases/) for introductory definitions.
+
+
+## Crates
+
+At the moment, leader election support is not exported from `kube`, and requires 3rd party crates. See [kube#485](https://github.com/kube-rs/kube/issues/485) for details.
+
+### kubert
+The [`kubert` crate](https://github.com/olix0r/kubert) via [olix0r/kubert](https://github.com/olix0r/kubert) contains several extensions on top of `kube`, and in particular a low-level `lease` module.
+
+```rust
+use kubert::lease::{ClaimParams, LeaseManager};
+let lease_api: Api<Lease> = Api::namespaced(client, namespace);
+let lease = LeaseManager::init(lease_api, name).await?;
+let claim = lease.ensure_claimed(&identity, &ClaimParams { ... }).await?;
+assert!(claim.is_current_for(&identity));
+```
+
+It is documented at [docs.rs/kubert/lease](https://docs.rs/kubert/latest/kubert/lease/index.html) and has a [large lease example](https://github.com/olix0r/kubert/blob/main/examples/lease.rs).
```

Comment:
If it helps to point this out, we use the kubert lease module in production, and people can consult the policy controller code:

https://github.com/linkerd/linkerd2/blob/1f4f4d417c6d06c3bd5a372fc75064f967117886/policy-controller/src/main.rs

Reply:
thanks, that is useful. have added a link.

https://github.com/kube-rs/website/pull/58/commits/bb7170183b97e215682e7807a651d9a6c529e78e